### PR TITLE
Update firebase.json

### PR DESCRIPTION
**Pull request by hellohuanlin**
Added the design doc https://docs.google.com/document/d/1XsaulkJA6_ZSpM7chkQLhQY25sQqhEMiqHMYzw8H85o/edit?resourcekey=0-_cUjF1c0iBvRLKfV-3gK5A

## Presubmit checklist
- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/master/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

**Comment by khanhnwin** [comment]
Thanks @hellohuanlin! Looks like the design doc isn't open to the public. Can you change the permissions on it, please?

**Comment by hellohuanlin** [comment]
@khanhnwin hmmm, strange, it should be open to the public already. could you try again?

**Comment by khanhnwin** [comment]
Hm, just tried again in an incognito window and it's still redirecting me to the Google login page. Is it possible that the link in firebase.json is the older locked-down link?

**Comment by hellohuanlin** [comment]
@khanhnwin very interesting, i added the resource key and it worked. will update the link

File: firebase.json [hunk 1]
```diff
@@ -276,6 +276,7 @@
       { "source": "/go/ios-camera-plugin-thread-model", "destination": "https://docs.google.com/document/d/1Znjp-BduXIb1Ix3JOE1YKyJi1Z0d6rB-2q6Lt7j8xSo/edit", "type": 301 },
       { "source": "/go/ios-platformview-backdrop-filter-blur", "destination": "https://docs.google.com/document/d/1V7Jc_RGaknrBBPPBBKB8lT7f3PKhYr8sin35MSMFAf4/edit?usp=sharing", "type": 301 },
       { "source": "/go/ios-platform-view-focus", "destination": "https://docs.google.com/document/d/1GG8VfYeuVtTQHiMWQXbrSG5HEUorzqU8IGbNySVJ-bY", "type": 301 },
+      { "source": "/go/ios-plugins-swift-migration", "destination": "https://docs.google.com/document/d/1XsaulkJA6_ZSpM7chkQLhQY25sQqhEMiqHMYzw8H85o", "type": 301 },
```
Comment (hellohuanlin) [inline firebase.json:279]: this works: 

https://docs.google.com/document/d/1XsaulkJA6_ZSpM7chkQLhQY25sQqhEMiqHMYzw8H85o/edit?resourcekey=0-_cUjF1c0iBvRLKfV-3gK5A

Comment (hellohuanlin) [reply firebase.json:279]: the other ones do not have this resource key tho

Comment (jmagman) [reply firebase.json:279]: go/drive-resource-keys#urls-with-resource-keys

Comment (khanhnwin) [reply firebase.json:279]: @hellohuanlin Yep, looks like the resource key did it! Once that's been added in the PR link we should be good to land. Thanks @jmagman!

**Review by khanhnwin — APPROVED** [review]

**Comment by khanhnwin** [comment]
Landed! Thanks @hellohuanlin!

**Pull request merged**